src/shrutiC/DuplicateNumbers.java

```diff
@@ -0,0 +1,18 @@
+package shrutiC;
+
+public class DuplicateNumbers {
+
+	public static void main(String[] args) {
+
+		int[] numArray = { 10, 12, 55, 32, 17, 12, 32 };
+
+		for (int index = 0; index < numArray.length; index++) {
```

Comment:
Do not write logic in main method.
{ 10, 12,32, 55, 32, 17, 12, 32 } try this input. code wont work as expected.